wordcloud/wordcloud.py

```diff
@@ -503,16 +508,17 @@ def process_text(self, text):
 
         flags = (re.UNICODE if sys.version < '3' and type(text) is unicode
                  else 0)
-        regexp = self.regexp if self.regexp is not None else r"\w[\w']+"
+        regexp = self.regexp if self.regexp is not None else r"(\w[\w']+|\w+)"
```

Comment:
Well it does overwrite the current behavior - you could say it does so by design. The current regex doesn't allow 1 character words, and you changed it so that it does allow one-character words. That makes the first part of the regex pretty superfluous, doesn't it?